README.md

```diff
@@ -29,3 +29,78 @@ More information: <a href="http://rosin-project.eu">rosin-project.eu</a>
 This project has received funding from the European Union’s Horizon 2020  
 research and innovation programme under grant agreement no. 732287. 
 
+
+
+
+# Welcome to the ROS End-Effector Action finder
+
+## Warning
+From 28-01-2020 the use_gui param gives an error because it is deprecated. This causes the sliders of joint 
+state publisher not shown. To solve : 
+```bash
+sudo apt install ros-kinetic-joint-state-publisher-gui
+```
+
+## Install Dependencies
+```bash
+sudo apt-get install ros-kinetic-moveit #moveit
+```
+
+## How to run for the Schunk Hand
```

Comment:
Can you elaborate a bit more on this?

e.g. 

> for the configuration phase THIS is going to happen. 
> The procedure will lead to the generation of THIS config files.
> 
> How to use the generator for another End-Effector?

I would add stuffs like this.

Reply:
Done in #14